src/Bookings.js

```diff
@@ -1,18 +1,49 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults";
 
 const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+  const [bookings, setBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIserror] = useState(false);
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/`)
+      .then(res => {
+        if (!res.ok) {
+          setIserror(true);
+        }
+        return res.json();
+      })
+      .then(data => {
+        setBookings(data);
+        console.log("all Bookings ", bookings);
+        setIsLoading(false);
+      })
+      .catch(console.error);
+  }, []);
+
+  const search = searchValue => {
+    if (searchValue) {
+      console.log("searchVal ", searchValue);
+      console.log("Bookings ", bookings);
+      const filteredBookings = bookings.filter(
+        booking =>
+          booking.firstName.toLowerCase().includes(searchValue.toLowerCase()) ||
+          booking.surname.toLowerCase().includes(searchValue.toLowerCase())
+      );
+      setBookings(filteredBookings);
```

Comment:
search is working well - it would be good to be able to "clear" a search and get back to the full list